packages/client/src/pages/Measurable/MeasurableEdit.re

```diff
@@ -25,23 +25,26 @@ module WithEditMutation = {
         labelOnDate: string,
         showDescriptionDate: string,
         labelProperty: string,
+        valueType: string,
       ) => {
     let date = showDescriptionDate == "TRUE" ? labelOnDate : "";
     let m =
       GraphQL.make(
         ~id,
         ~input={
           "name": name,
-          "labelCustom": labelCustom |> E.O.some,
-          "labelProperty": labelProperty |> E.O.some,
-          "labelOnDate": date |> Js.Json.string |> E.O.some,
+          "labelCustom": labelCustom |> Rationale.Option.some,
```

Comment:
Why did you change this? I'm trying to stay away from the use of ``Rationale`` directly, unless we want to dramatically change use of the E library.

Reply:
Because it is not readable.